### PR TITLE
fix(l10n): fix input type direction to match Gmail

### PR DESCRIPTION
**Pull request by vladikoff**
Fixes https://github.com/mozilla/fxa-content-server/issues/4056

### CODE REVIEW

**Comment by vladikoff** [comment]
@vbudhram could you check when you have a sec.

If you set Hebrew or any other RTL locale, it should work the same way as gmail.com

**Comment by vbudhram** [comment]
@vladikoff LGTM 👍, r+

As a side note, when testing this found that there are several areas where the text is not displayed in `rtl` format, is this ok? In your picture above, there area inconsistent layouts too.

File: app/styles/modules/_input-row.scss [hunk 1]
```diff
@@ -65,9 +65,9 @@
     }
 
     html[dir='rtl'] & {
-      direction: rtl;
+      text-align: right;
+      direction: ltr;
       padding: 0  $input-left-right-padding 0 0;
-      unicode-bidi: bidi-override;
```
Comment (TDA) [inline app/styles/modules/_input-row.scss]: nit: wont this cause a regression with the special characters? IIRC that was the reason this was even added in the first place @vladikoff

Comment (vladikoff) [inline app/styles/modules/_input-row.scss]: no, with the new rules it's almost like normal `ltr` but aligned right.

**Pull request merged**